package.json

```
@@ -50,9 +50,11 @@
     "fix:lint": "eslint . --ext .js,.ts,.tsx,.mjs --fix",
     "generate:api:client": "SWAGGER_CODEGEN_CLI_VERSION=3.0.32 ./scripts/generate_api_client.sh",
     "generate:api:client:silicon": "SWAGGER_CODEGEN_CLI_VERSION=3.0.32 ./scripts/generate_api_client_silicon.sh",
-    "ios:prod": "react-native run-ios --scheme PassCulture-Production",
-    "ios:staging": "react-native run-ios --scheme PassCulture-Staging",
-    "ios:testing": "react-native run-ios --scheme PassCulture-Testing",
+    "preios": "./scripts/check_xcode_version.sh",
```

```suggestion
    "preios": "./scripts/check_xcode_version.sh",
```
on pourrait imaginer une commande `yarn doctor` ou `yarn checkup` qui vérifie les problèmes connus sur le set up de dev au delà d'xcode

---

attention, le nom de commit suggère que cette vérification est appelée avant d'appeler `yarn ios:*` ce qui n'est pas le cas

---

> le nom de commit suggère que cette vérification est appelée avant d'appeler yarn ios:* ce qui n'est pas le cas

pourquoi ça ne serait pas le cas ?